catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -240,6 +239,77 @@ const WorkDetails: FunctionComponent<Props> = ({ work }: Props) => {
 
   const Content = () => (
     <>
+      {showHoldingsOnWork && holdings.length > 0 && (
+        <WorkDetailsSection headingText="Holdings" isInArchive={isInArchive}>
+          {holdings.map((holding, i) => (
+            <div key={i}>
```

Comment:
Do we have a better key?

Reply:
Don't think we do, the only thing we know we get on a holding is type (which will be "Holdings" for everything) and enumerations, which can be an empty list